### DISCUSSION TITLE
Will argo rollouts support kuma service mesh for traffic splitting inside the k8s cluster?

### DISCUSSION THREAD

**Discussion by sathyanarayanan-sekar** [argoproj/argo-rollouts#3734] [Q&A]
Hey Team,

Will argo rollouts work with kuma service mesh for implementing canary, blue green deployment strategies for k8s deployments and also with traffic splitting to the service. Any examples of argo rollouts with kuma might be useful for me.

Thanks

**Comment by kostis-codefresh** [comment]
Isn't this already possible via the [Gateway API support](https://konghq.com/blog/product-releases/kuma-1-6-0-and-kong-mesh-1-7-0-released-with-kubernetes-gateway-api-support-aws-acm-integration-and-more) and the [plugin ](https://github.com/argoproj-labs/rollouts-plugin-trafficrouter-gatewayapi) ?

Have you tested this combination?
